packages/components/pie-checkbox/test/visual/pie-checkbox.spec.ts

```diff
@@ -65,10 +61,8 @@ componentVariants.forEach((variant) => test(`should render all prop variations f
         const propKeyValues = `
             checked: ${testComponent.propValues.checked},
             disabled: ${testComponent.propValues.disabled},
-            label: ${testComponent.propValues.label ? 'with label' : 'no label'},
             status: ${testComponent.propValues.status},
-            indeterminate: ${testComponent.propValues.indeterminate},
-            assistiveText: ${testComponent.propValues.assistiveText ? 'with assistive text' : 'no assistive text'}`;
```

Comment:
I removed assistiveText from this tests to to decrease the amount of information on one screenshot since assistive text covered on another screenshot